indico/web/client/js/react/components/PersonLinkField.jsx

```diff
@@ -281,6 +287,7 @@ function PersonLinkField({
     values.forEach(p => {
       p.name = formatName(p);
       p.roles = roles.filter(x => x.default).map(x => x.name);
+      getPluginObjects('onAddPersonLink').forEach(f => f(p));
```

Comment:
I'd do `const hooks =  getPluginObjects('onAddPersonLink');` outside the `values.forEach()` loop and then just run `hooks.forEach(...)` here. No need to build that list every time...